flow360/component/flow360_params/flow360_params.py

```diff
@@ -694,6 +726,16 @@ def to_solver(self, params: Flow360Params, **kwargs) -> Boundaries:
         """
         return super().to_solver(params, **kwargs)
 
+    @classmethod
+    def _get_widgets(cls) -> dict[str, str]:
+        return {
+            "additionalProperties/velocity/value": "vector3",
+            "additionalProperties/Velocity/value": "vector3",
+            "additionalProperties/velocityDirection/value": "vector3",
+            "additionalProperties/translationVector": "vector3",
+            "additionalProperties/axisOfRotation": "vector3",
+        }
+
```

Comment:
write a comment what is this for

Reply:
Added comment, renamed schema-specific functions